.github/workflows/nix.yml

```diff
@@ -35,6 +35,8 @@ jobs:
     runs-on: ubuntu-24.04
     timeout-minutes: 30
     steps:
+      # Installing rust toolchain to setup envs for rust-cache action even if using Nix
+      - run: rustup toolchain install '1.82.0' --profile minimal # selfup {"extract":"\\d[^']+","replacer":["rustc", "--version"], "nth": 2}
```

Comment:
This change doesn't fix rust-cache action rustc version

```
  .. Environment considered:
    - Rust Version: 1.85.0 x86_64-unknown-linux-gnu (4d91de4e48198da2e33413efdcd9cd2cc0c46688)
```

Reply:
https://github.com/Swatinem/rust-cache/blob/27b8ea9368cf428f0bfe41b0876b1a7e809d9844/src/config.ts#L362-L370